packages/api-explorer/src/components/DocMarkdown/DocMarkdown.tsx

```diff
@@ -56,7 +56,7 @@ export const DocMarkdown: FC<DocMarkdownProps> = ({ source, specKey }) => {
   const handleClick = (e: BaseSyntheticEvent) => {
     if (
       e.target.tagName === 'A' &&
-      e.target.origin === window.location.origin
+      e.target.pathname.startsWith(`/${specKey}`)
```

Comment:
consider instantiating a history singleton in `index.ts` and exporting it:

```
import { createBrowserHistory } from 'history'
export const browserHistory = createBrowserHistory()
<Router history={browserHistory}>
  ...
</Router>
```

then you can

```
import { browserHistory } from '.'
...
browserHistory.matchPath(e.target.pathname, `/${specKey}`)
```

this could give you a little more type safety down the line, if e.g. you wanted to change specKey to an enum from a routes object.